schema.sql

```diff
@@ -22,6 +24,15 @@ CREATE TABLE author(
     last_name varchar(50)
 );
 
+CREATE TABLE genre(
+    genre_id INT IDENTITY(1,1) PRIMARY KEY,
+    name varchar(50)
+);
+
+CREATE TABLE music_album(
+    album_id INT IDENTITY(1,1) PRIMARY KEY,
+    on_spotify boolean,
+
```

Comment:
Kindly close this CREATE TABLE statement.
```sql
CREATE TABLE music_album(
    album_id INT IDENTITY(1,1) PRIMARY KEY,
    on_spotify boolean);
``

Reply:
@enoqueJonas Thanks for the second eye :eyes: on my work. :muscle: